package.json

```diff
@@ -69,7 +69,7 @@
   "scripts": {
     "knit": "knit",
     "build": "knit build --skip-preflight --verbose",
-    "start": "knit server",
+    "start": "rm data/schema.json&&rm data/version.json&&knit schema&&knit server",
```

Comment:
A thought, `knit schema` should overwrite the existing files right? 
a command like `knit schema && knit server` might be better because then it will fail to start if you can't get the schema, but it doesn't delete the schema so you can still edit the command and run with the existing files.